src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+
+    public Figure getDefaultFigure() {
+        Figure circle = new Circle("white", 10);
+        return circle;
+    }
+
+    public Figure getRandomFigure() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        String color = colorSupplier.getRandomColor();
+        Figure [] figures = new Figure[5];
```

Comment:
Consider defining the number of figures as a constant, e.g., `private static final int FIGURE_COUNT = 5;`, to avoid magic numbers in your code.